source/simulator/solver.cc

```diff
@@ -193,15 +193,15 @@ namespace aspect
       public:
         /**
          * Constructor.
-         * @param mp_matrix Matrix approximating S to be used in the inner solve
-         * @param mp_preconditioner The preconditioner for @p mp_matrix
+         * @param pressure_laplace_operator Laplace operator on the pressure space. This is how we choose to discretize (BC^{-1}B^T).
+         * @param mp_preconditioner The preconditioner for @p pressure_laplace_operator
          * @param solver_tolerance The relative solver tolerance for the inner solve
          * @param inverse_lumped_mass_matrix Lumped mass matrix associated with the velocity block
          * @param system_matrix Sparse block matrix storing the Stokes system of the form
          * [A B^T
          *  B 0].
          */
-        WeightedBFBT(const LinearAlgebra::SparseMatrix &mp_matrix,
+        WeightedBFBT(const LinearAlgebra::SparseMatrix &presure_laplace_operator,
```

Comment:
we typically use the name "operator" if it is something implicitly defined, and ``matrix`` if it is an actual matrix (which is is here).